src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -7,7 +7,8 @@ import Divider from '@mui/material/Divider';
 import Typography from '@mui/material/Typography';
 import { parse } from 'query-string';
 import React, { useCallback, useEffect } from 'react';
-import { ReactFlowProvider } from 'react-flow-renderer';
+import { ReactFlowProvider } from 'reactflow';
+import 'reactflow/dist/style.css';
```

Comment:
I don't think we need these styles everywhere, just in the react flow canvas file.